src/core/qgsogrutils.cpp

```diff
@@ -998,6 +998,8 @@ QgsFields QgsOgrUtils::stringToFields( const QString &string, QTextCodec *encodi
 QStringList QgsOgrUtils::cStringListToQStringList( char **stringList )
 {
   QStringList strings;
+  if ( !stringList )
```

Comment:
What about moving that _above_ the QStringList strings to save a few CPU cycles? :)